parse.go

```diff
@@ -541,17 +540,18 @@ func readNativeFrames(d *dicomio.Decoder, parsedData *element.DataSet, frameChan
 		dicomlog.Vprintf(1, "ERROR finding bits allocated.")
 		return nil, 0, err
 	}
-	bitsAllocated := int(b.MustGetUInt16())
+	bitsAllocated := int(b.MustGetInt())
```

Comment:
hmm for now we know that we will never overflow `int32` since DICOM ints are max 32 bits afaik, and these values will never overflow in particular. Still `NativeFrame` should be updated to use explicit int64 for these values...though the data can still be `int`s...